cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -639,6 +639,9 @@
 
     public static final String METRICS_WRITER_EXTENSIONS_DIR = "metrics.writer.extensions.dir";
 
+    // Metrics writer configuration file path.
+    public static final String METRICS_WRITER_CONFIG_PATH = "metrics.writer.config.path";
```

Comment:
This belongs to the extension? Shouldn't be defined in CDAP

Reply:
fixed